File: Leetcode/JS/Leetcode268-丢失的数字.js
/**
 * @param {number[]} nums
 * @return {number}
 */
 var missingNumber = function(nums) {
    var res = 0;
    for(let i = 0;i < nums.length;i++){
        res ^= nums[i];
    }
    for(let i = 0;i <= nums.length;i++){
        res ^= i;
    }
    return res;
};

File: Leetcode/Rust/Leetcode268-丢失的数字.rs
impl Solution {
    pub fn missing_number(nums: Vec<i32>) -> i32 {
        let (mut res,n) = (0,nums.len());
        for num in nums.iter(){
            res ^= num;
        }
        for i in 0..=n{
            res ^= i as i32;
        }
        return res;
    }
}